### PR TITLE
chore: remove storybook from url

### PR DESCRIPTION
**Pull request by jakemhiller**
### Overview

Removes /storybook from the github pages url

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: packages/styleguide/package.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "start": "yarn storybook",
     "storybook": "start-storybook -p 6006 -c .storybook --ci --quiet",
-    "build-storybook": "NODE_ENV=production build-storybook --quiet -c .storybook -o ../../dist/storybook",
-    "build-storybook-alpha": "NODE_ENV=production build-storybook --quiet -c .storybook -o ../../dist/storybook"
+    "build-storybook": "NODE_ENV=production build-storybook --quiet -c .storybook -o ../../dist/docs",
```
Comment (jakemhiller) [inline packages/styleguide/package.json:9]: once this is merged i'll configure gh pages to serve from the `/docs` directory

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@44.6.3-alpha.709960.0<br/></pre>

File: dist/static/storybook/index.html [hunk 2]
```diff
@@ -11,7 +11,13 @@
 </head>
 
 <body>
-  <script>window.location.replace('/storybook')</script>
+  <script>
+    // Preserve existing links to /storybook with a janky redirect
```
Comment (JoshuaKGoldberg) [inline dist/static/storybook/index.html:15]: 😄

**Review by JoshuaKGoldberg — APPROVED** [review]
Thanks!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@44.6.3-alpha.0f34f1.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6143713c897f7a30650df23a--gamut-preview.netlify.app>https://6143713c897f7a30650df23a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6143713c897f7a30650df23a>Deploy Logs</a></p>

**Pull request merged**